### PR TITLE
 Created receipt class

### PR DESCRIPTION
**Pull request by ShriGrover**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -28,4 +28,6 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation 'com.google.android.material:material:1.0.0-rc01'
     api 'com.theartofdev.edmodo:android-image-cropper:2.8.+'
+    implementation 'androidx.recyclerview:recyclerview:1.1.0-beta01'
+    implementation 'androidx.cardview:cardview:1.0.0'
```
Comment (adayyli) [inline app/build.gradle]: nit: should be grouped together. implementation should all go before testImplementation

File: app/src/main/java/com/billbreaker/MainActivity.java [hunk 2]
```diff
@@ -52,6 +59,10 @@ public void onClick(View view) {
         });
     }
 
+    private void populateReceipts() {
+        // receipt = receiptDatabase.getAllReceipts(); TODO uncomment this when ada updates receipt database
```
Comment (adayyli) [inline app/src/main/java/com/billbreaker/MainActivity.java]: receipts (not receipt)

File: app/src/main/java/com/billbreaker/Receipt.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    public long getTimestamp() {
+        return timestamp;
+    }
+
+    public void setTimestamp(long timestamp) {
+        this.timestamp = timestamp;
+    }
+}
```
Comment (adayyli) [inline app/src/main/java/com/billbreaker/Receipt.java:30]: newline EOF

File: app/src/main/java/com/billbreaker/MainActivity.java (hunk 2 above)
Comment (ShriGrover) [reply app/src/main/java/com/billbreaker/MainActivity.java]: done

File: app/build.gradle (hunk 1 above)
Comment (ShriGrover) [reply app/build.gradle]: done

**Pull request merged**